src/main/java/uk/gov/justice/digital/hmpps/csr/api/dto/DetailDto.kt

```diff
@@ -23,28 +22,28 @@ data class ShiftOvertimeDto @JsonCreator constructor(
         @JsonProperty("end")
         val end: Long,
 
-        @ApiModelProperty(required = true, value = "Type of shift", position = 4, example = "overtime")
+        @ApiModelProperty(required = true, value = "Type of detail", position = 4, example = "overtime")
         @JsonProperty("type")
         val type: String,
 
-        @ApiModelProperty(required = true, value = "Detail of shift task", position = 5, example = "Canteen watch")
-        @JsonProperty("task")
-        val task: String
+        @ApiModelProperty(required = true, value = "Detail activity", position = 5, example = "Canteen watch")
+        @JsonProperty("activity")
+        val activity: String
 ) {
         companion object {
 
-                fun from(it: ShiftOvertime): ShiftOvertimeDto {
-                        return ShiftOvertimeDto(
+                fun from(it: Detail): DetailDto {
+                        return DetailDto(
                                 it.date,
                                 it.detailStartTimeInSeconds,
                                 it.detailEndTimeInSeconds,
                                 ShiftType.OVERTIME.name,
```

Comment:
If this is now a generic detail dto, we shoud hardcode the shift type to overtime elsewhere

Reply:
https://github.com/ministryofjustice/cmd-csr-procs/blob/master/Notification_Shift_Detail/GET_MODIFIED_DETAIL.sql#L60

See what mons comes up with but he can do something like that I think ^^^^